README.rst

```diff
@@ -84,7 +84,10 @@ Docker
 
 A Dockerfile is also provided; see the comments on the top of the file for
 configuration instructions. The docker image is regularly built and
-published at ``ghcr.io/jelmer/xandikos``.
+published at ``ghcr.io/jelmer/xandikos``. See 
+``examples/docker-compose.yml`` and the 
+`man page <http://manpages.ubuntu.com/manpages/impish/man1/xandikos.1.html>`_
```

Comment:
I'd rather not link to an external site - especially as this link is tied to a particular ubuntu version. Let's publish the manpage on the xandikos site and then link to that. I'd be great if you could submit a PR to https://github.com/xandikos/homepage but I'm also happy to take a look at that (I think it's fairly easy).

Reply:
I've added a copy on https://www.xandikos.org/manpage.html (may take up to 30 minutes to propagate)

Reply:
Awesome, I've updated the PR to use that url instead. 👍